src/pages/disabled/Disabled.tsx

```diff
@@ -2,21 +2,31 @@ import { Classes, NonIdealState } from '@blueprintjs/core';
 import { IconNames } from '@blueprintjs/icons';
 import classNames from 'classnames';
 import * as React from 'react';
+import { useLoaderData } from 'react-router';
 
 interface DisabledProps {
   reason?: string;
 }
 
-const Disabled: React.FC<DisabledProps> = ({ reason }) => (
-  <div className={classNames('NoPage', Classes.DARK)}>
-    <NonIdealState
-      icon={IconNames.ERROR}
-      title="Disabled"
-      description={`The Source Academy has been disabled${
-        reason ? ` for this reason: ${reason}` : ''
-      }.`}
-    />
-  </div>
-);
+const Disabled: React.FC<DisabledProps> = () => {
```

Comment:
It seems that we can simply remove `DisabledProps` here. Is there a way we can still check that the component is rendered properly, with the correct reason?

Reply:
yup i updated the testcases to reflect that. also #2506 might be relevant